src/components/tiles/table/table-tile.tsx

```diff
@@ -253,13 +278,22 @@ const TableToolComponent: React.FC<ITileProps> = observer(function TableToolComp
   });
 
   useEffect(() => {
+    if (gridRef.current?.element) {
+      setGridElement(gridRef.current.element);
+    }
     if (containerRef.current) {
       // override the CSS variables controlling selection color for linked tables
       const dataGrid = containerRef.current.getElementsByClassName("rdg")[0] as HTMLDivElement | undefined;
       dataGrid?.style.setProperty("--header-selected-background-color", "rgba(0,0,0,0)");
       dataGrid?.style.setProperty("--row-selected-background-color", "rgba(0,0,0,0)");
     }
-  });
+  }, [gridRef, containerRef]);
+
+  // useEffect(() => {
+  //   if (gridRef.current?.element) {
+  //     setGridElement(gridRef.current.element);
+  //   }
+  // }, [gridRef]);
```

Comment:
Either delete this commented out code or add a note why it is useful to keep it around.